registration/settings.py

```diff
@@ -20,10 +25,10 @@
 # See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "django-insecure-5*-kc^mwgvy)2g(8@lj$q6=ulwbj2b7!^8vyk28nvgaqmmmsp5"
+SECRET_KEY = os.getenv('DJANGO_SECRET_KEY')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = os.getenv('DJANGO_DEBUG')
```

Comment:
A default value would be a good idea here I think, defaulting to the production setting (debug off) should be fine.
```suggestion
DEBUG = os.getenv('DJANGO_DEBUG', False)
```